astropy/coordinates/tests/test_sky_coord.py

```diff
@@ -903,7 +903,7 @@ def test_wcs_methods(mode, origin):
     from astropy.utils.data import get_pkg_data_contents
     from astropy.wcs.utils import pixel_to_skycoord
 
-    header = get_pkg_data_contents('../../wcs/tests/maps/1904-66_TAN.hdr', encoding='binary')
+    header = get_pkg_data_contents('../../wcs/tests/data/maps/1904-66_TAN.hdr', encoding='binary')
```

Comment:
This makes me wonder if shared data files should be in an upper level `data` above both `wcs` and `coordinates`.

Reply:
Not a bad idea - but maybe worth doing in another PR since I'm not sure what the correct location should be.